### PR TITLE
l10n-zhtw Traditional Chinese Pack for SonarQube-8.0

### PR DESCRIPTION
**Pull request by timlee**
sonar-l10n-zh-tw-plugin-1.0 support SonarQube-8.0. Traditional Chinese Language Pack for SonarQube 8.0

### CODE REVIEW

**Comment by timlee** [comment]
Hi Ann,  
I have changed the file name of l10nzh-tw.properties to l10nzhtw.properties. 
The plugin key l10nzhtw is also registered in the master list. 
Please see if there any problems. 
Thank you.

File: l10nzhtw.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+description=SonarQube Traditional Chinese Pack
+homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zhtw
+archivedVersions=
+publicVersions=1.0
+
+defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
+defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+
+1.0.description=Traditional Chinese Pack Support SonarQube 8.0
+1.0.sqVersions=[7.9,LATEST]
```
Comment (ganncamp) [inline l10nzhtw.properties]: Your description says 8.0 and your compatibility says 7.9+. Please make them match. :-)

& sorry I didn't notice this before.

File: l10nzhtw.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
+category=Localization
+description=SonarQube Traditional Chinese Pack
```
Comment (ganncamp) [inline l10nzhtw.properties]: Please update the description to mention incompatibility with the l10n-zh pack.

Comment (timlee) [reply l10nzhtw.properties]: I have revised compatibility in sqVersion and mention incompatibility with the l10n-zh pack. Please see if there is any problem. Thank you.

File: l10nzhtw.properties [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+description=SonarQube Traditional Chinese Language Pack (not compatible with the “Chinese Pack” which uses Simplified Chinese)
+homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zhtw
+archivedVersions=
+publicVersions=1.0
+
+defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
+defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+
+1.0.description=SonarQube Traditional Chinese Pack
+1.0.sqVersions=[6.0,LATEST]
```
Comment (ganncamp) [inline l10nzhtw.properties]: Did you mean to start compatibility with 6.0? That's no longer supported by the Marketplace. You _can_ go as low as 6.7 (altho we'll be dropping it from the Marketplace in a few days).
Did you mean 8.0 instead?

Comment (timlee) [reply l10nzhtw.properties]: My SonarQube version is 7.5 and it works with Traditional Chinese plugin. Can I use [7.5, LATEST] instead?

Comment (timlee) [reply l10nzhtw.properties]: I have updated it to [7.5, LATEST], please let me know if there is any problem.

**Pull request merged**